repose-aggregator/components/services/atom-feed-service/atom-feed-service-api/src/main/java/org/openrepose/nodeservice/atomfeed/AuthenticatedRequestFactory.java

```diff
@@ -29,11 +27,11 @@
     /**
      * Mutates a request to adhere to some authentication scheme compatible with the Atom service.
      *
-     * @param atomFeedUrlConnection The unauthenticated feed {@link URLConnection} with the Atom service.
-     * @param context               A context object which contains information related to the request.
-     * @return The {@link URLConnection} with authentication mutations applied.
+     * @param feedReadRequest A mutable container for request data that will be sent to the Atom service.
```

Comment:
Yeah, this should be handled with exceptions, returning null as a failure is bad business.